polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -55,6 +55,8 @@ export interface TooltipProps {
    * @default '1'
    */
   borderRadius?: BorderRadius;
+  /** Override on the default z-index of 400 */
+  zIndexOverride?: number;
```

Comment:
Initially I thought no since this would mirror the Popover implementation, though it does seem ideal to enforce the usage of tokens.

Reply:
As in making the `zIndexOverride` into the token types, but this currently matches the Popover prop exactly 😄 https://github.com/Shopify/polaris/blob/8fc0fefdfcedd92830639d040a042055a0105f71/polaris-react/src/components/Popover/Popover.tsx#L55

Reply:
I think it would require some reworking since the PositionedOverlay accepts a number, which would impact all Popover usages:

https://github.com/Shopify/polaris/blob/256c4d4e7dfb6f441903db8cbbf0314f6258d876/polaris-react/src/components/PositionedOverlay/PositionedOverlay.tsx#L42

Reply:
Another reason is that in the overrides seen in the Web repo, if we want to enable this prop for them then it would need to accept any number value as some calculations look fairly custom:

https://github.com/Shopify/web/blob/14cb6d154ccd9dfc0e2e763e22d2d5dfb077ff38/app/shared/dev-ui/components/Console/Console.scss#L15-L22